base/usage/v1alpha/usage.proto

```diff
@@ -63,27 +63,29 @@ message Session {
   string name = 1 [(google.api.field_behavior) = OUTPUT_ONLY];
   // Resource UUID
   string uid = 2 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Owner UUID of the instance, can also be considered as instance UUID
+  string owner_uid = 3 [(google.api.field_behavior) = REQUIRED];
```

Comment:
I suggest we move the index of `owner_uid` to `13` and keep others be the same. Because the usage server may need to receive message from both new and old usage client. We'd better preserve the backward-compatibility.

Reply:
done